addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+| `no_proxy`                       | Optional          | A comma-separated list of hostnames, IP addresses, or IP ranges in CIDR format that should not use a proxy                                                                                        |
+
+## Usage Example
+
+### Getting started with kpack using the `kp` cli
+
+> Note: This guide assumes that you have provided the `kp_default_repository` values during install.
+> If you do no want to set those values, you can follow the [kubectl flow](#getting-started-with-kpack-using-kubectl) below.
+> If you forgot to provide them at install, you can update your values and then update the install using the `tanzu` cli.
+
+> Note: This guide assumes that [`kp`](https://github.com/vmware-tanzu/kpack-cli) has been downloaded. It can be installed using [homebrew](https://github.com/vmware-tanzu/homebrew-kpack-cli), by downloading from the [github release](https://github.com/vmware-tanzu/kpack-cli/releases), or through a [Docker image](https://hub.docker.com/r/kpack/kp).
+
+1. Log in to the `kp_default_repository` locally.
+
+    ```bash
+    docker login <REGISTRY-HOSTNAME> -u <REGISTRY-USER>
+    ```
+
+   > Note: The `<REGISTRY-HOSTNAME>` must be the registry prefix for its corresponding registry
+   > - For [Dockerhub](https://hub.docker.com/) this should be `https://index.docker.io/v1/`. `kp` also offers a simplified way to create a Dockerhub secret with a `--dockerhub` flag.
+   > - For [GCR](https://cloud.google.com/container-registry/) this should be `gcr.io`. If you use GCR then the username can be `_json_key` and the password can be the JSON credentials you get from the GCP UI (under `IAM -> Service Accounts` create an account or edit an existing one and create a key with type JSON). `kp` also offers a simplified way to create a gcr secret with a `--gcr` flag.
+
+2. Create a ClusterStore.
+
+   A store resource is a repository of [buildpacks](http://buildpacks.io/) packaged
+   in [buildpackages](https://buildpacks.io/docs/buildpack-author-guide/package-a-buildpack/) that can be used by kpack
+   to build OCI images. Later in this tutorial, you will reference this store in a Builder configuration.
+
+   We recommend starting with buildpacks from the [paketo project](https://github.com/paketo-buildpacks). The example
+   below pulls in java buildpack from the paketo project. If you would like to use a different language you can select a different buildpack.
+
+    ```
+    kp clusterstore save default -b gcr.io/paketo-buildpacks/java
+    ```
+
+   > Note: Buildpacks are packaged and distributed as buildpackages which are OCI images available on a Docker registry. Buildpackages for other languages are available from [paketo](https://github.com/paketo-buildpacks).
+
```

Comment:
I trying this guide, I needed to run the step. Should that be added here?
```
kp config default-repository
```

Reply:
did you configure the kp_default_repository during install? You shouldn't have to use that command if you did